.code-samples.meilisearch.yaml

```diff
@@ -840,15 +876,6 @@ search_parameter_guide_show_ranking_score_1: |-
     .execute()
     .await
     .unwrap();
-search_parameter_guide_show_ranking_score_details_1: |-
-  let results: SearchResults<Movie> = client
-    .index("movies")
-    .search()
-    .with_query("dragon")
-    .with_show_ranking_score_details(true)
-    .execute()
-    .await
-    .unwrap();
```

Comment:
Ok my bad, I was confused with the conflicts, you were not supposed to delete this chunk of code

Reply:
sorry about that, I believe I recharged everything appropriately